downstream/modules/platform/proc-accessing-rpm-repositories-for-locally-mounted-dvd.adoc

```diff
@@ -52,4 +52,4 @@ https://www.redhat.com/security/data/fd431d51.txt [Could not resolve host:
 www.redhat.com]
 ----
 
-In order to set up a repository see link:https://access.redhat.com/solutions/3776721[Need to set up yum repository for locally-mounted DVD on Red Hat Enterprise Linux 8].
+For further detail on setting up a repository see link:https://access.redhat.com/solutions/3776721[Need to set up yum repository for locally-mounted DVD on Red Hat Enterprise Linux 8].
```

Comment:
Suggestion: add this as an "Additional Resources" section
```suggestion
.Additional resources

For further detail on setting up a repository see link:https://access.redhat.com/solutions/3776721[Need to set up yum repository for locally-mounted DVD on Red Hat Enterprise Linux 8].
```

Reply:
I think this would normally be a good formatting recommendation. However, I'm thinking about removing this module entirely and just linking to the knowledgebase article in a different module. If you look at the original text, it says the steps were copied from the article. Imo, better to link than to duplicate information in our documentation. What do you think?

Reply:
Sounds good to me! I trust your judgement.